### DISCUSSION TITLE
Network version ?

### DISCUSSION THREAD

**Discussion by spikemixture** [hluk/CopyQ#2167] [Ideas]
I love CopyQ and use it daily on 2 desktops PC's. 
OR 
Can I link these two version so they both see and save my clipboarded data to the same place?
I have a NAS which both PC's see so I can move the appropriate file there. 
That is if I know how to point the 2 ClipQ PC's/versions to that location !

**Comment by hluk** [comment]
This is supported only with the synchronization plugin: https://copyq.readthedocs.io/en/latest/synchronize.html

**Reply by spikemixture** [reply]
Thanks hluk, 
I have gone to the that section and pointed each individual tab on each of the 2 PC's to the individual tabs own folder on my NAS. e.g. D:\BACKUPS\copyQ\clipboard (default) and D:\BACKUPS\copyQ\computer (created tab)

What should happen as both PC's do not show the same 'clips' ?

**Reply by spikemixture** [reply]
No answer ?

**Reply by hluk** [reply]
How different are the synced tabs? It is possible that due to CopyQ configuration or access rights (or maybe other shared file system config) the items would be different.
